src/main/java/frc/robot/RobotContainer.java

```diff
@@ -125,13 +133,15 @@ public RobotContainer() {
         runSysidTrigger = new Trigger(() -> getInput().getRunSysId());
         shootSpeakerTrigger = new Trigger(() -> getInput().getShouldShootSpeaker());
         shootAmpTrigger = new Trigger(() -> getInput().getShouldShootAmp());
+        reZeroClimbTrigger = new Trigger(() -> getInput().getReZeroClimbers());
```

Comment:
Not sure we need a button to re-zero the climber. Why not just a switch on the shuffleboard? It shouldn't need to happen all that often.